### ISSUE TITLE
about training the RIFE_HDv2

### ISSUE THREAD

**Issue by tqyunwuxin** [hzwer/ECCV2022-RIFE#147]
hello , I  train the RIFE_HD successfully ,but training RIFE_HDv2 failed. the optical flow labels is produced by  liteflownet with  Vimeo90K dataset.

to train RIFE_HDv2, I use " 3e-4*mul" in function "get_learning_rate",    batch size 16, and loss "loss_G = loss_l1 + loss_cons + loss_ter" ,  optimG "weight_decay=1e-5"

Is there anything else to pay special attention to？
it is a little difficult to converge because the output of flownet is four channels？

**Comment by hzwer** [comment]
Hi, I set weight_decay=1e-3 and it should be more stable.
If it can still not converge, please set get_learning_rate to "1e-4*mul".
